### ISSUE TITLE
TG desktop crashes on linux when trying to play video

### ISSUE THREAD

**Issue by igorvolnyi** [telegramdesktop/tdesktop#24747]
### Steps to reproduce

1. Open any chat or channel where there is some video
2. Try to play
3. Telegram crashes

### Expected behaviour

Obviously play video without SEGFAULT

### Actual behaviour

TG desktop shut down with Segmentation Fault

### Operating system

Arch Linux x86_64

### Version of Telegram Desktop

4.0.2-2

### Installation source

Other (unofficial) source

### Logs

[2022.07.03 00:17:49] Launched version: 4000002, install beta: [TRUE], alpha: 0, debug mode: [FALSE]
[2022.07.03 00:17:49] Executable dir: /usr/bin/, name: telegram-desktop
[2022.07.03 00:17:49] Initial working dir: /home/igor/
[2022.07.03 00:17:49] Working dir: /home/igor/.local/share/TelegramDesktop/
[2022.07.03 00:17:49] Command line: /bin/telegram-desktop --
[2022.07.03 00:17:49] Executable path before check: /usr/bin/telegram-desktop
[2022.07.03 00:17:49] Logs started
[2022.07.03 00:17:49] Launcher filename: telegramdesktop.desktop
[2022.07.03 00:17:49] Getting DE, XDG_CURRENT_DESKTOP: 'KDE'
[2022.07.03 00:17:49] Getting DE, XDG_SESSION_DESKTOP: 'KDE'
[2022.07.03 00:17:49] Getting DE, DESKTOP_SESSION: 'plasma'
[2022.07.03 00:17:49] Getting DE via WM: 'KWin'
[2022.07.03 00:17:49] Getting DE, GNOME_DESKTOP_SESSION_ID: ''
[2022.07.03 00:17:49] Getting DE, KDE_FULL_SESSION: 'true'
[2022.07.03 00:17:49] DE: KDE
[2022.07.03 00:17:49] Connecting local socket to /run/user/1001/c615bf515373ed8699da19f597688127-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2022.07.03 00:17:49] This is the only instance of Telegram, starting server and app...
[2022.07.03 00:17:49] Moved logging from '/home/igor/.local/share/TelegramDesktop/log_start0.txt' to '/home/igor/.local/share/TelegramDesktop/log.txt'!
[2022.07.03 00:17:49] Primary screen DPI: 96
[2022.07.03 00:17:49] System tray available: [TRUE]
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.07.03 00:17:49] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.03 00:17:49] Icon theme: Arc
[2022.07.03 00:17:49] Fallback icon theme: hicolor
[2022.07.03 00:17:49] App Info: reading settings...
[2022.07.03 00:17:49] App Info: reading encrypted settings...
[2022.07.03 00:17:50] Lang Info: Loaded cached, keys: 4134
[2022.07.03 00:17:50] OpenAL Logging Level: (not set)
[2022.07.03 00:17:50] Audio Playback Devices: Built-in Audio Analog Stereo
[2022.07.03 00:17:50] Audio Playback Default Device: Built-in Audio Analog Stereo
[2022.07.03 00:17:50] Audio Capture Devices: Built-in Audio Analog Stereo;Monitor of Built-in Audio Analog Stereo
[2022.07.03 00:17:50] Audio Capture Default Device: Built-in Audio Analog Stereo
[2022.07.03 00:17:50] Using Unity launcher counter.
[2022.07.03 00:17:50] App Info: reading accounts info...
[2022.07.03 00:17:50] App Info: reading encrypted info...
[2022.07.03 00:17:50] App Info: reading map...
[2022.07.03 00:17:50] App Info: reading encrypted map...
[2022.07.03 00:17:50] App Info: reading encrypted user settings...
[2022.07.03 00:17:50] App Info: encrypted user settings read.
[2022.07.03 00:17:50] App Info: reading encrypted mtp data...
[2022.07.03 00:17:50] MTP Info: read keys, current: 3, to destroy: 0
[2022.07.03 00:17:50] Map read time: 4
[2022.07.03 00:17:50] App Info: reading encrypted mtp config...
[2022.07.03 00:17:50] Export Info: Destroy top bar by controller removal.
[2022.07.03 00:17:51] OpenGL Profile: Compatibility.
[2022.07.03 00:17:51] OpenGL Renderer: NVIDIA GeForce GT 640/PCIe/SSE2
[2022.07.03 00:17:51] OpenGL Vendor: NVIDIA Corporation
[2022.07.03 00:17:51] OpenGL Version: 4.6.0 NVIDIA 470.129.06
[2022.07.03 00:17:51] OpenGL Extensions: GL_ARB_fragment_program_shadow, GL_NV_texture_compression_vtc, GL_ARB_shader_precision, GL_NV_robustness_video_memory_purge, GL_ARB_texture_rgb10_a2ui, GL_NV_shader_thread_group, GL_ARB_texture_env_add, GL_EXT_blend_minmax, GL_EXT_vertex_array, GL_NV_gpu_program4_1, GL_ARB_explicit_uniform_location, GL_ATI_texture_float, GL_EXT_multiview_timer_query, GL_ARB_program_interface_query, GL_ATI_draw_buffers, GL_ARB_point_parameters, GL_NV_alpha_to_coverage_dither_control, GL_ARB_pipeline_statistics_query, GL_ARB_buffer_storage, GL_ARB_shader_objects, GL_S3_s3tc, GL_EXT_blend_equation_separate, GL_ARB_multi_bind, GL_NV_vertex_array_range, GL_NV_path_rendering, GL_ARB_geometry_shader4, GL_NV_texture_barrier, GL_ARB_depth_clamp, GL_EXT_secondary_color, GL_EXT_shader_image_load_store, GL_EXT_point_parameters, GL_ARB_texture_border_clamp, GL_ARB_sparse_buffer, GL_ARB_map_buffer_range, GL_NV_gpu_multicast, GL_EXT_shadow_funcs, GL_EXT_polygon_offset_clamp, GL_EXT_packed_depth_stencil, GL_ARB_vertex_type_2_10_10_10_rev, GL_ARB_compressed_texture_pixel_storage, GL_EXT_semaphore, GL_ARB_blend_func_extended, GL_NV_gpu_program_fp64, GL_EXT_direct_state_access, GL_EXT_texture_filter_anisotropic, GL_NV_packed_depth_stencil, GL_AMD_seamless_cubemap_per_texture, GL_EXT_framebuffer_blit, GL_ARB_compute_shader, GL_ARB_shading_language_packing, GL_ARB_shader_bit_encoding, GL_ARB_texture_barrier, GL_ARB_vertex_shader, GL_EXT_texture_env_dot3, GL_EXT_semaphore_fd, GL_ARB_point_sprite, GL_ARB_ES3_compatibility, GL_ARB_timer_query, GL_ARB_indirect_parameters, GL_NV_vertex_program2_option, GL_ARB_seamless_cube_map, GL_ARB_shader_ballot, GL_ARB_half_float_vertex, GL_EXT_framebuffer_multisample, GL_EXT_texture_sRGB_decode, GL_NV_compute_program5, GL_NV_multisample_coverage, GL_KHR_debug, GL_NV_query_resource_tag, GL_NV_draw_vulkan_image, GL_NV_float_buffer, GL_ARB_clip_control, GL_EXT_texture_cube_map, GL_ARB_map_buffer_alignment, GL_EXT_rescale_normal, GL_NV_bindless_texture, GL_EXT_abgr, GL_NV_shader_buffer_load, GL_ARB_transform_feedback2, GL_NVX_conditional_render, GL_KHR_blend_equation_advanced, GL_ARB_texture_query_lod, GL_EXT_separate_specular_color, GL_ARB_texture_compression_bptc, GL_ARB_compute_variable_group_size, GL_ARB_texture_filter_anisotropic, GL_ARB_texture_float, GL_NV_vertex_program1_1, GL_NVX_gpu_memory_info, GL_NV_vdpau_interop, GL_EXT_texture_sRGB, GL_SGIS_generate_mipmap, GL_ARB_depth_buffer_float, GL_NV_timeline_semaphore, GL_EXT_texture_mirror_clamp, GL_ARB_uniform_buffer_object, GL_EXT_blend_subtract, GL_ARB_gpu_shader_fp64, GL_EXT_fog_coord, GL_EXT_texture_compression_s3tc, GL_EXT_separate_shader_objects, GL_NV_vdpau_interop2, GL_OVR_multiview, GL_NV_texgen_reflection, GL_NV_draw_texture, GL_ARB_vertex_buffer_object, GL_OVR_multiview2, GL_NV_primitive_restart, GL_NV_fragment_program2, GL_NV_depth_buffer_float, GL_ARB_compatibility, GL_ARB_shader_image_size, GL_ARB_transform_feedback_overflow_query, GL_EXT_texture_compression_rgtc, GL_EXT_transform_feedback2, GL_EXT_blend_func_separate, GL_EXT_texture_env_add, GL_NV_point_sprite, GL_NV_shader_atomic_float, GL_ARB_internalformat_query2, GL_ARB_texture_storage, GL_ARB_texture_cube_map, GL_NV_blend_square, GL_ARB_internalformat_query, GL_SGIX_depth_texture, GL_ARB_vertex_type_10f_11f_11f_rev, GL_EXT_texture_env_combine, GL_ARB_texture_storage_multisample, GL_KHR_parallel_shader_compile, GL_NV_ES3_1_compatibility, GL_ARB_conditional_render_inverted, GL_EXT_gpu_shader4, GL_KTX_buffer_region, GL_NVX_progress_fence, GL_ARB_framebuffer_object, GL_SGIS_texture_lod, GL_EXT_packed_pixels, GL_KHR_shader_subgroup, GL_EXT_texture_buffer_object, GL_NV_texture_shader, GL_NV_vertex_buffer_unified_memory, GL_ARB_shading_language_include, GL_ARB_draw_buffers, GL_ARB_texture_compression, GL_ARB_half_float_pixel, GL_ARB_texture_gather, GL_ARB_get_program_binary, GL_ARB_vertex_program, GL_ARB_vertex_attrib_64bit, GL_EXT_texture_shared_exponent, GL_ARB_texture_non_power_of_two, GL_NV_parameter_buffer_object2, GL_EXT_texture3D, GL_EXT_provoking_vertex, GL_ARB_texture_rg, GL_EXT_framebuffer_object, GL_ARB_shader_texture_image_samples, GL_ARB_texture_view, GL_ARB_debug_output, GL_ARB_shader_texture_lod, GL_EXT_Cg_shader, GL_KHR_robustness, GL_NV_half_float, GL_ARB_conservative_depth, GL_ARB_texture_swizzle, GL_ARB_occlusion_query2, GL_IBM_texture_mirrored_repeat, GL_NV_vertex_program, GL_ARB_clear_buffer_object, GL_ARB_polygon_offset_clamp, GL_ARB_ES3_1_compatibility, GL_ARB_sampler_objects, GL_ARB_vertex_array_bgra, GL_NV_bindless_multi_draw_indirect_count, GL_ARB_fragment_layer_viewport, GL_NV_register_combiners2, GL_NV_uniform_buffer_unified_memory, GL_EXT_memory_object, GL_ARB_gpu_shader_int64, GL_ARB_depth_texture, GL_ARB_vertex_array_object, GL_ARB_texture_buffer_range, GL_EXT_draw_range_elements, GL_NV_parameter_buffer_object, GL_NV_copy_image, GL_ARB_texture_rectangle, GL_EXT_depth_bounds_test, GL_ARB_multitexture, GL_EXT_texture_storage, GL_EXT_multiview_texture_multisample, GL_EXT_texture_shadow_lod, GL_ARB_direct_state_access, GL_ARB_window_pos, GL_NV_gpu_program4, GL_ARB_copy_image, GL_ARB_multisample, GL_EXT_stencil_wrap, GL_NV_gpu_program5, GL_ARB_base_instance, GL_EXT_stencil_two_side, GL_ARB_shading_language_100, GL_NV_copy_depth_to_color, GL_NV_fragment_program_option, GL_EXT_texture_integer, GL_NV_texture_shader2, GL_NV_shader_thread_shuffle, GL_ARB_shading_language_420pack, GL_EXT_timer_query, GL_KHR_robust_buffer_access_behavior, GL_EXT_framebuffer_multisample_blit_scaled, GL_ARB_draw_elements_base_vertex, GL_EXT_window_rectangles, GL_ARB_shader_storage_buffer_object, GL_ARB_fragment_shader, GL_ARB_color_buffer_float, GL_ARB_query_buffer_object, GL_ARB_shader_image_load_store, GL_ARB_ES2_compatibility, GL_ARB_sync, GL_ARB_enhanced_layouts, GL_EXT_texture_lod, GL_NV_register_combiners, GL_NV_pixel_data_range, GL_ARB_texture_compression_rgtc, GL_ARB_texture_query_levels, GL_ARB_explicit_attrib_location, GL_NV_shader_subgroup_partitioned, GL_NV_query_resource, GL_ARB_texture_buffer_object, GL_ARB_framebuffer_no_attachments, GL_ARB_shader_atomic_counters, GL_EXT_texture_edge_clamp, GL_SUN_slice_accum, GL_EXT_x11_sync_object, GL_NV_depth_clamp, GL_EXT_pixel_buffer_object, GL_ARB_texture_env_crossbar, GL_ARB_robust_buffer_access_behavior, GL_NV_vertex_attrib_integer_64bit, GL_EXT_bindable_uniform, GL_NV_bindless_multi_draw_indirect, GL_AMD_multi_draw_indirect, GL_ARB_copy_buffer, GL_ARB_transform_feedback3, GL_NV_explicit_multisample, GL_NV_occlusion_query, GL_NV_texture_rectangle_compressed, GL_ARB_spirv_extensions, GL_ARB_clear_texture, GL_ARB_transpose_matrix, GL_NV_texture_rectangle, GL_ARB_shader_clock, GL_ARB_seamless_cubemap_per_texture, GL_NV_texture_shader3, GL_ARB_provoking_vertex, GL_EXT_texture_compression_latc, GL_ARB_draw_buffers_blend, GL_ARB_vertex_attrib_binding, GL_NV_transform_feedback2, GL_ARB_bindless_texture, GL_ARB_tessellation_shader, GL_ARB_derivative_control, GL_ARB_texture_env_combine, GL_EXT_texture_sRGB_R8, GL_ARB_shader_group_vote, GL_EXT_multi_draw_arrays, GL_ARB_parallel_shader_compile, GL_ARB_shadow, GL_ARB_framebuffer_sRGB, GL_NV_transform_feedback, GL_ARB_texture_multisample, GL_ARB_stencil_texturing, GL_ARB_texture_cube_map_array, GL_NV_fence, GL_EXT_gpu_program_parameters, GL_ARB_imaging, GL_ARB_draw_indirect, GL_EXT_draw_buffers2, GL_SGIX_shadow, GL_ARB_separate_shader_objects, GL_ARB_pixel_buffer_object, GL_NV_vertex_program2, GL_NVX_nvenc_interop, GL_ARB_texture_env_dot3, GL_NV_shader_storage_buffer_object, GL_EXT_draw_instanced, GL_NV_conditional_render, GL_EXT_framebuffer_sRGB, GL_ARB_sample_shading, GL_EXT_texture_compression_dxt1, GL_ARB_transform_feedback_instanced, GL_ARB_texture_buffer_object_rgb32, GL_NV_feature_query, GL_NV_fog_distance, GL_EXT_texture_object, GL_EXT_packed_float, GL_EXT_vertex_attrib_64bit, GL_ARB_texture_mirrored_repeat, GL_NV_internalformat_sample_query, GL_EXT_geometry_shader4, GL_ARB_shader_subroutine, GL_ARB_occlusion_query, GL_NV_texture_multisample, GL_ARB_invalidate_subdata, GL_ARB_sparse_texture, GL_EXT_bgra, GL_EXT_import_sync_object, GL_ARB_fragment_coord_conventions, GL_KHR_context_flush_control, GL_NV_texture_env_combine4, GL_NV_light_max_exponent, GL_ARB_cull_distance, GL_ARB_multi_draw_indirect, GL_ARB_shader_atomic_counter_ops, GL_EXT_texture_swizzle, GL_EXT_vertex_array_bgra, GL_NV_command_list, GL_EXTX_framebuffer_mixed_formats, GL_NV_framebuffer_multisample_coverage, GL_NV_multisample_filter_hint, GL_EXT_blend_color, GL_ATI_texture_mirror_once, GL_NV_ES1_1_compatibility, GL_NV_geometry_shader4, GL_EXT_memory_object_fd, GL_ARB_robustness, GL_ARB_draw_instanced, GL_ARB_instanced_arrays, GL_IBM_rasterpos_clip, GL_KHR_no_error, GL_EXT_compiled_vertex_array, GL_ARB_viewport_array, GL_NV_gpu_shader5, GL_ARB_ES3_2_compatibility, GL_ARB_shader_draw_parameters, GL_ARB_texture_stencil8, GL_NV_shader_atomic_counters, GL_EXT_texture_lod_bias, GL_NV_vertex_program3, GL_ARB_arrays_of_arrays, GL_ARB_get_texture_sub_image, GL_ARB_fragment_program, GL_ARB_texture_mirror_clamp_to_edge, GL_NV_vertex_array_range2, GL_NV_blend_equation_advanced, GL_EXT_texture_array, GL_ARB_gl_spirv, GL_NV_gpu_program5_mem_extended, GL_ARB_gpu_shader5, GL_NV_fragment_program, GL_EXT_shader_integer_mix
[2022.07.03 00:17:51] OpenGL: Disable on NVIDIA driver on Linux.
[2022.07.03 00:17:51] OpenGL: [FALSE] (OverlayWidget)
[2022.07.03 00:17:51] Notification daemon product name: Plasma
[2022.07.03 00:17:51] Notification daemon vendor name: KDE
[2022.07.03 00:17:51] Notification daemon version: 5.25.2
[2022.07.03 00:17:51] Notification daemon specification version: 1.2
[2022.07.03 00:17:51] Notification daemon capabilities: body, body-hyperlinks, body-markup, body-images, icon-static, actions, persistence, inline-reply, x-kde-urls, x-kde-origin-name, x-kde-display-appname, inhibitions
[2022.07.03 00:17:52] Message Info: bad message notification received (error_code 33) for msg_id = 7115856595139756680, seq_no = 42
[2022.07.03 00:17:52] Message Info: bad message notification received, msgId 7115856595139756680, error_code 33
[2022.07.03 00:17:52] API Warning: not loaded minimal channel applied.
[2022.07.03 00:17:53] API Warning: not loaded minimal channel applied.
[2022.07.03 00:17:53] API Warning: not loaded minimal channel applied.
[2022.07.03 00:17:53] API Warning: not loaded minimal channel applied.
[2022.07.03 00:17:53] API Warning: not loaded minimal channel applied.
[2022.07.03 00:17:53] API Warning: not loaded minimal channel applied.
[2022.07.03 00:22:02] RPC Error: request 494 got fail with code 400, error USER_NOT_PARTICIPANT
[2022.07.03 00:23:38] Message Info: bad message notification received (error_code 33) for msg_id = 7115858082114613796, seq_no = 222
[2022.07.03 00:23:38] Message Info: bad message notification received, msgId 7115858082114613796, error_code 33
[2022.07.03 00:36:15] API Warning: not loaded minimal channel applied.
[2022.07.03 00:36:28] Message Info: bad message notification received (error_code 33) for msg_id = 7115861388800411608, seq_no = 364
[2022.07.03 00:39:16] API Warning: not loaded minimal channel applied.
[2022.07.03 00:41:10] API Warning: not loaded minimal channel applied.
[2022.07.03 00:41:11] API Warning: not loaded minimal channel applied.
[2022.07.03 01:13:59] RPC Error: request 2156 got fail with code 400, error USER_NOT_PARTICIPANT
[2022.07.03 01:17:18] Audio Info: recreating audio device and reattaching the tracks

**Comment by Aokromes** [comment]
plz try downloading official version from https://desktop.telegram.org

**Comment by igorvolnyi** [comment]
UPDATE: does not crash after disabling Hardware Acceleration for playing video

**Comment by ilya-fedin** [comment]
@john-preston maybe move hardware acceleration into experimental settings or hide it completely?

**Comment by igorvolnyi** [comment]
Maybe it's an issue of older Nvidia drivers (470 version)

**Comment by ilya-fedin** [comment]
There weren't reached any stability with hardware acceleration, that's why it's disabled by default. But people still trying to enable it and file bug reports :(

**Comment by igorvolnyi** [comment]
> @john-preston maybe move hardware acceleration into experimental settings or hide it completely?

I think it would be better to just explicitly mark it as "experimental" or "unstable" and give a link to some documentation explaining what's going on and what can go wrong.

**Comment by ilya-fedin** [comment]
> and give a link to some documentation explaining what's going on and what can go wrong

That's not a common practice in tdesktop 😅

**Comment by J-E-Mc** [comment]
I'm also getting this error with hardware acceleration enabled, but the logs seem to say nothing about the error before the program crashes
```
[2022.07.18 18:51:54] Launched version: 4000002, install beta: [FALSE], alpha: 0, debug mode: [FALSE]
[2022.07.18 18:51:54] Executable dir: /usr/bin/, name: telegram-desktop
[2022.07.18 18:51:54] Initial working dir: /home/mcard/.local/share/TelegramDesktop/
[2022.07.18 18:51:54] Working dir: /home/mcard/.local/share/TelegramDesktop/
[2022.07.18 18:51:54] Command line: /usr/bin/telegram-desktop -noupdate -tosettings
[2022.07.18 18:51:54] Executable path before check: /usr/bin/telegram-desktop
[2022.07.18 18:51:54] Logs started
[2022.07.18 18:51:54] Launcher filename: telegramdesktop.desktop
[2022.07.18 18:51:54] Getting DE, XDG_CURRENT_DESKTOP: ''
[2022.07.18 18:51:54] Getting DE, XDG_SESSION_DESKTOP: ''
[2022.07.18 18:51:54] Getting DE, DESKTOP_SESSION: ''
[2022.07.18 18:51:54] Getting DE via WM: 'i3'
[2022.07.18 18:51:54] Getting DE, GNOME_DESKTOP_SESSION_ID: ''
[2022.07.18 18:51:54] Getting DE, KDE_FULL_SESSION: ''
[2022.07.18 18:51:54] DE: Other
[2022.07.18 18:51:54] Connecting local socket to /run/user/1000/3037c7aba36cd1a945a62303f68217e6-{87A94AB0-E370-4cde-98D3-ACC110C5967D}...
[2022.07.18 18:51:54] This is the only instance of Telegram, starting server and app...
[2022.07.18 18:51:54] Moved logging from '/home/mcard/.local/share/TelegramDesktop/log_start0.txt' to '/home/mcard/.local/share/TelegramDesktop/log.txt'!
[2022.07.18 18:51:54] Primary screen DPI: 96
[2022.07.18 18:51:54] System tray available: [FALSE]
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAOpenSansRegular.ttf' loaded 'DAOpenSansRegular'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAOpenSansRegularItalic.ttf' loaded 'DAOpenSansRegularItalic'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAVazirRegular.ttf' loaded 'DAVazirRegular'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAOpenSansSemiboldAsBold.ttf' loaded 'DAOpenSansSemibold'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAOpenSansSemiboldItalicAsBold.ttf' loaded 'DAOpenSansSemiboldItalic'
[2022.07.18 18:51:54] Font: from ':/gui/fonts/DAVazirMediumAsBold.ttf' loaded 'DAVazirMedium'
[2022.07.18 18:51:54] Icon theme: hicolor
[2022.07.18 18:51:54] Fallback icon theme: hicolor
[2022.07.18 18:51:54] App Info: reading settings...
[2022.07.18 18:51:54] App Info: reading encrypted settings...
[2022.07.18 18:51:54] Lang Info: Loaded cached, keys: 4132
[2022.07.18 18:51:54] Lang Info: Loaded cached, keys: 2081
[2022.07.18 18:51:54] Lang Error: Unexpected tag (key 'lng_call_box_status_group:count')
[2022.07.18 18:51:54] OpenAL Logging Level: (not set)
[2022.07.18 18:51:54] Audio Playback Devices: Audio Adapter (Unitek Y-247A) Analog Stereo;GP104 High Definition Audio Controller Digital Stereo (HDMI);Built-in Audio Digital Stereo (IEC958);Built-in Audio Analog Stereo
[2022.07.18 18:51:54] Audio Playback Default Device: Audio Adapter (Unitek Y-247A) Analog Stereo
[2022.07.18 18:51:54] Audio Capture Devices: Audio Adapter (Unitek Y-247A) Mono;Monitor of GP104 High Definition Audio Controller Digital Stereo (HDMI);Monitor of Audio Adapter (Unitek Y-247A) Analog Stereo;Monitor of Built-in Audio Digital Stereo (IEC958);Monitor of Built-in Audio Analog Stereo;Built-in Audio Analog Stereo
[2022.07.18 18:51:54] Audio Capture Default Device: Audio Adapter (Unitek Y-247A) Mono
[2022.07.18 18:51:54] Not using Unity launcher counter.
[2022.07.18 18:51:54] App Info: reading accounts info...
[2022.07.18 18:51:54] App Info: reading encrypted info...
[2022.07.18 18:51:54] App Info: reading map...
[2022.07.18 18:51:54] App Info: reading encrypted map...
[2022.07.18 18:51:54] App Info: reading encrypted user settings...
[2022.07.18 18:51:54] App Info: encrypted user settings read.
[2022.07.18 18:51:54] App Info: reading encrypted mtp data...
[2022.07.18 18:51:54] MTP Info: read keys, current: 3, to destroy: 0
[2022.07.18 18:51:54] Map read time: 0
[2022.07.18 18:51:54] App Info: reading encrypted mtp config...
[2022.07.18 18:51:54] Export Info: Destroy top bar by controller removal.
[2022.07.18 18:51:54] OpenGL Profile: Compatibility.
[2022.07.18 18:51:54] OpenGL Renderer: NVIDIA GeForce GTX 1070/PCIe/SSE2
[2022.07.18 18:51:54] OpenGL Vendor: NVIDIA Corporation
[2022.07.18 18:51:54] OpenGL Version: 4.6.0 NVIDIA 515.57
[2022.07.18 18:51:54] OpenGL Extensions: GL_EXT_separate_specular_color, GL_ARB_shader_precision, GL_KHR_shader_subgroup, GL_NV_framebuffer_multisample_coverage, GL_ARB_texture_env_add, GL_EXT_framebuffer_multisample, GL_EXT_texture_array, GL_NV_gpu_program4, GL_NV_internalformat_sample_query, GL_NV_texture_rectangle, GL_SGIX_depth_texture, GL_ARB_compatibility, GL_NV_robustness_video_memory_purge, GL_ARB_fragment_shader, GL_NV_vertex_array_range, GL_ARB_texture_filter_anisotropic, GL_EXT_vertex_array, GL_NV_draw_vulkan_image, GL_NV_gpu_shader5, GL_ARB_ES2_compatibility, GL_EXT_texture_env_dot3, GL_ARB_derivative_control, GL_EXT_packed_depth_stencil, GL_NV_copy_image, GL_EXT_texture_filter_anisotropic, GL_EXT_depth_bounds_test, GL_NV_memory_object_sparse, GL_NV_ES1_1_compatibility, GL_ARB_vertex_buffer_object, GL_ATI_texture_float, GL_ARB_texture_gather, GL_ARB_occlusion_query2, GL_ARB_copy_image, GL_ARB_point_parameters, GL_ARB_vertex_program, GL_NV_bindless_multi_draw_indirect, GL_KHR_debug, GL_NV_geometry_shader4, GL_NV_half_float, GL_ARB_sparse_texture, GL_ARB_vertex_type_10f_11f_11f_rev, GL_NV_fragment_program_option, GL_NV_blend_equation_advanced_coherent, GL_NV_stereo_view_rendering, GL_EXT_geometry_shader4, GL_EXT_texture_compression_rgtc, GL_EXT_texture_cube_map, GL_NV_blend_equation_advanced, GL_NV_fragment_shader_interlock, GL_ARB_texture_filter_minmax, GL_EXT_import_sync_object, GL_ARB_shading_language_packing, GL_EXT_blend_color, GL_NVX_blend_equation_advanced_multi_draw_buffers, GL_NV_packed_depth_stencil, GL_EXT_framebuffer_blit, GL_NV_fragment_program2, GL_ARB_texture_env_combine, GL_NV_shader_atomic_float, GL_ARB_vertex_attrib_64bit, GL_ARB_depth_buffer_float, GL_NV_path_rendering_shared_edge, GL_ARB_shader_storage_buffer_object, GL_EXT_texture_edge_clamp, GL_NV_primitive_restart, GL_NV_float_buffer, GL_ARB_seamless_cubemap_per_texture, GL_ARB_texture_mirror_clamp_to_edge, GL_NV_register_combiners, GL_NV_clip_space_w_scaling, GL_NV_texture_multisample, GL_ARB_multi_draw_indirect, GL_EXT_window_rectangles, GL_EXT_texture_swizzle, GL_ARB_texture_view, GL_NV_texture_shader, GL_ARB_shader_objects, GL_EXT_texture_lod, GL_EXT_direct_state_access, GL_ARB_indirect_parameters, GL_NV_command_list, GL_NV_vertex_program, GL_NV_texture_barrier, GL_NV_shader_atomic_fp16_vector, GL_NV_gpu_program5_mem_extended, GL_EXT_texture_sRGB_R8, GL_ARB_texture_compression_bptc, GL_EXT_texture_integer, GL_ARB_tessellation_shader, GL_EXT_shader_integer_mix, GL_IBM_rasterpos_clip, GL_ARB_geometry_shader4, GL_EXT_texture_compression_latc, GL_NVX_progress_fence, GL_ARB_framebuffer_object, GL_ARB_invalidate_subdata, GL_ARB_pipeline_statistics_query, GL_NV_fragment_coverage_to_color, GL_AMD_seamless_cubemap_per_texture, GL_EXT_shader_image_load_formatted, GL_ARB_pixel_buffer_object, GL_EXT_raster_multisample, GL_SGIS_texture_lod, GL_ARB_get_program_binary, GL_EXT_draw_range_elements, GL_KHR_robust_buffer_access_behavior, GL_ARB_ES3_compatibility, GL_NV_shader_thread_group, GL_ARB_sync, GL_ARB_shading_language_include, GL_ARB_robust_buffer_access_behavior, GL_ARB_draw_indirect, GL_EXT_shadow_funcs, GL_NV_path_rendering, GL_ARB_texture_stencil8, GL_EXT_x11_sync_object, GL_NV_sample_locations, GL_NV_query_resource, GL_NV_bindless_multi_draw_indirect_count, GL_EXT_transform_feedback2, GL_ARB_texture_buffer_range, GL_ARB_clear_buffer_object, GL_NV_multisample_coverage, GL_ARB_map_buffer_alignment, GL_AMD_multi_draw_indirect, GL_ARB_compute_shader, GL_ARB_clear_texture, GL_ARB_spirv_extensions, GL_NV_draw_texture, GL_ARB_texture_swizzle, GL_NV_vertex_program3, GL_ARB_internalformat_query, GL_NV_shader_thread_shuffle, GL_SUN_slice_accum, GL_ARB_texture_query_lod, GL_NV_vdpau_interop, GL_NV_sample_mask_override_coverage, GL_NV_transform_feedback, GL_NV_uniform_buffer_unified_memory, GL_ARB_vertex_shader, GL_AMD_vertex_shader_viewport_index, GL_ARB_debug_output, GL_NV_depth_clamp, GL_NV_alpha_to_coverage_dither_control, GL_NV_vertex_program2, GL_KHR_blend_equation_advanced, GL_ARB_direct_state_access, GL_ARB_texture_border_clamp, GL_EXT_compiled_vertex_array, GL_ARB_texture_rg, GL_EXT_provoking_vertex, GL_ARB_transform_feedback3, GL_ARB_gpu_shader_int64, GL_EXT_rescale_normal, GL_EXT_draw_instanced, GL_ARB_vertex_attrib_binding, GL_NV_conditional_render, GL_ARB_shading_language_100, GL_EXT_blend_subtract, GL_ARB_program_interface_query, GL_ARB_draw_buffers, GL_NV_fog_distance, GL_ARB_polygon_offset_clamp, GL_ARB_multisample, GL_ARB_compressed_texture_pixel_storage, GL_EXT_packed_pixels, GL_ARB_texture_mirrored_repeat, GL_NV_texture_shader2, GL_EXT_secondary_color, GL_NV_feature_query, GL_ARB_half_float_vertex, GL_NV_texture_env_combine4, GL_ARB_texture_compression, GL_ARB_sample_shading, GL_ARB_texture_multisample, GL_ARB_fragment_coord_conventions, GL_NV_conservative_raster_pre_snap_triangles, GL_ARB_sparse_buffer, GL_NV_register_combiners2, GL_ATI_draw_buffers, GL_NV_shader_atomic_counters, GL_NV_shader_atomic_float64, GL_ARB_copy_buffer, GL_NVX_gpu_memory_info, GL_NV_gpu_multicast, GL_ARB_multi_bind, GL_EXT_framebuffer_sRGB, GL_ARB_base_instance, GL_EXT_texture_shadow_lod, GL_EXT_framebuffer_multisample_blit_scaled, GL_EXT_separate_shader_objects, GL_NV_gpu_program5, GL_ARB_texture_rectangle, GL_ARB_viewport_array, GL_NV_memory_attachment, GL_NV_vertex_buffer_unified_memory, GL_ARB_vertex_array_bgra, GL_EXT_gpu_shader4, GL_ARB_texture_cube_map_array, GL_ARB_draw_instanced, GL_ARB_ES3_1_compatibility, GL_ARB_shader_subroutine, GL_ARB_texture_compression_rgtc, GL_ARB_transform_feedback_overflow_query, GL_EXT_abgr, GL_ARB_shader_texture_lod, GL_ARB_transpose_matrix, GL_S3_s3tc, GL_OVR_multiview2, GL_NV_gpu_program_fp64, GL_EXT_vertex_array_bgra, GL_ARB_internalformat_query2, GL_EXT_multiview_texture_multisample, GL_EXT_texture_object, GL_ARB_draw_elements_base_vertex, GL_ARB_depth_clamp, GL_ARB_transform_feedback2, GL_KHR_parallel_shader_compile, GL_ARB_separate_shader_objects, GL_ARB_vertex_array_object, GL_EXT_texture_lod_bias, GL_EXT_texture_buffer_object, GL_NV_timeline_semaphore, GL_NV_conservative_raster, GL_ARB_fragment_program_shadow, GL_NV_fragment_program, GL_EXT_bgra, GL_ARB_conservative_depth, GL_NV_vertex_program2_option, GL_EXT_texture_storage, GL_ARB_half_float_pixel, GL_KHR_no_error, GL_ARB_framebuffer_no_attachments, GL_EXT_point_parameters, GL_NV_bindless_texture, GL_ARB_query_buffer_object, GL_EXT_packed_float, GL_EXT_texture_compression_dxt1, GL_ARB_compute_variable_group_size, GL_NV_shader_subgroup_partitioned, GL_ARB_shader_image_size, GL_EXT_semaphore_fd, GL_ARB_texture_query_levels, GL_EXT_shader_image_load_store, GL_NVX_conditional_render, GL_NV_explicit_multisample, GL_ARB_multitexture, GL_ARB_conditional_render_inverted, GL_ARB_enhanced_layouts, GL_ARB_robustness, GL_ARB_stencil_texturing, GL_ARB_parallel_shader_compile, GL_EXT_gpu_program_parameters, GL_EXT_stencil_wrap, GL_IBM_texture_mirrored_repeat, GL_KTX_buffer_region, GL_NV_point_sprite, GL_NV_texture_shader3, GL_NV_vdpau_interop2, GL_ARB_texture_buffer_object, GL_ARB_texture_env_crossbar, GL_ATI_texture_mirror_once, GL_EXT_framebuffer_object, GL_NV_multisample_filter_hint, GL_NV_pixel_data_range, GL_ARB_blend_func_extended, GL_ARB_clip_control, GL_ARB_depth_texture, GL_ARB_texture_cube_map, GL_ARB_fragment_program, GL_ARB_shader_viewport_layer_array, GL_KHR_robustness, GL_NV_shader_buffer_load, GL_EXT_timer_query, GL_ARB_transform_feedback_instanced, GL_ARB_color_buffer_float, GL_OVR_multiview, GL_EXT_semaphore, GL_ARB_instanced_arrays, GL_ARB_fragment_shader_interlock, GL_ARB_texture_barrier, GL_ARB_sampler_objects, GL_ARB_texture_float, GL_ARB_get_texture_sub_image, GL_ARB_ES3_2_compatibility, GL_ARB_vertex_type_2_10_10_10_rev, GL_EXTX_framebuffer_mixed_formats, GL_EXT_texture_compression_s3tc, GL_ARB_imaging, GL_NV_blend_minmax_factor, GL_KHR_blend_equation_advanced_coherent, GL_NV_fill_rectangle, GL_ARB_arrays_of_arrays, GL_ARB_shader_draw_parameters, GL_ARB_post_depth_coverage, GL_ARB_shader_bit_encoding, GL_ARB_texture_rgb10_a2ui, GL_ARB_texture_storage, GL_NV_fence, GL_NVX_nvenc_interop, GL_EXT_fog_coord, GL_NV_transform_feedback2, GL_ARB_seamless_cube_map, GL_NV_parameter_buffer_object, GL_ARB_gl_spirv, GL_ARB_buffer_storage, GL_EXT_texture_sRGB, GL_NV_blend_square, GL_NV_texture_compression_vtc, GL_ARB_shader_ballot, GL_EXT_blend_equation_separate, GL_ARB_gpu_shader5, GL_EXT_sparse_texture2, GL_NV_light_max_exponent, GL_NV_viewport_array2, GL_NV_shader_storage_buffer_object, GL_ARB_shading_language_420pack, GL_ARB_explicit_attrib_location, GL_NV_texture_rectangle_compressed, GL_EXT_texture_env_add, GL_EXT_vertex_attrib_64bit, GL_NV_compute_program5, GL_EXT_texture_mirror_clamp, GL_SGIX_shadow, GL_EXT_polygon_offset_clamp, GL_EXT_memory_object_fd, GL_ARB_fragment_layer_viewport, GL_NV_gpu_program4_1, GL_ARB_map_buffer_range, GL_ARB_shader_atomic_counters, GL_EXT_memory_object, GL_ARB_timer_query, GL_ARB_shader_clock, GL_SGIS_generate_mipmap, GL_ARB_texture_env_dot3, GL_NV_conservative_raster_dilate, GL_NV_query_resource_tag, GL_ARB_explicit_uniform_location, GL_EXT_post_depth_coverage, GL_ARB_shader_group_vote, GL_EXT_texture_sRGB_decode, GL_NV_viewport_swizzle, GL_ARB_point_sprite, GL_ARB_occlusion_query, GL_ARB_sparse_texture2, GL_EXT_texture3D, GL_NV_texgen_reflection, GL_ARB_shader_atomic_counter_ops, GL_ARB_shader_texture_image_samples, GL_ARB_sparse_texture_clamp, GL_EXT_blend_func_separate, GL_NV_parameter_buffer_object2, GL_ARB_draw_buffers_blend, GL_EXT_texture_filter_minmax, GL_ARB_texture_non_power_of_two, GL_EXT_pixel_buffer_object, GL_NV_shader_atomic_int64, GL_ARB_uniform_buffer_object, GL_NV_ES3_1_compatibility, GL_NV_vertex_program1_1, GL_ARB_shadow, GL_EXT_stencil_two_side, GL_EXT_multi_draw_arrays, GL_NV_vertex_attrib_integer_64bit, GL_ARB_bindless_texture, GL_EXT_bindable_uniform, GL_EXT_texture_env_combine, GL_EXT_draw_buffers2, GL_NV_framebuffer_mixed_samples, GL_KHR_context_flush_control, GL_EXT_texture_shared_exponent, GL_ARB_framebuffer_sRGB, GL_ARB_shader_image_load_store, GL_ARB_window_pos, GL_EXT_blend_minmax, GL_NV_geometry_shader_passthrough, GL_ARB_sample_locations, GL_NV_vertex_array_range2, GL_ARB_cull_distance, GL_ARB_texture_storage_multisample, GL_ARB_provoking_vertex, GL_NV_occlusion_query, GL_ARB_gpu_shader_fp64, GL_EXT_multiview_timer_query, GL_ARB_texture_buffer_object_rgb32, GL_AMD_vertex_shader_layer, GL_NV_copy_depth_to_color, GL_NV_depth_buffer_float, GL_EXT_Cg_shader
[2022.07.18 18:51:54] OpenGL: Disable on NVIDIA driver on Linux.
[2022.07.18 18:51:54] OpenGL: [FALSE] (OverlayWidget)
[2022.07.18 18:51:54] Notification daemon product name: dunst
[2022.07.18 18:51:54] Notification daemon vendor name: knopwob
[2022.07.18 18:51:54] Notification daemon version: 1.9.0
[2022.07.18 18:51:54] Notification daemon specification version: 1.2
[2022.07.18 18:51:54] Notification daemon capabilities: actions, body, body-hyperlinks, icon-static, synchronous, private-synchronous, x-canonical-private-synchronous, x-dunst-stack-tag, body-markup
[2022.07.18 18:51:56] Audio Info: recreating audio device and reattaching the tracks
[2022.07.18 18:52:00] Audio Info: Closing audio playback device.
[2022.07.18 18:53:12] Audio Info: recreating audio device and reattaching the tracks
```

**Comment by gouchi** [comment]
@igorvolnyi this issue should be renamed from `TG desktop crashes on linux when trying to play video ` to something `TG desktop crashes on linux when trying to play video with hardware acceleration enabled`.

**Comment by ilya-fedin** [comment]
It's not hardware acceleration but hardware decoding. By the term 'acceleration' usually rendering APIs are meant (OpenGL, D3D, Vulkan).

**Comment by igorvolnyi** [comment]
Looks like this issue has very low priority. If at all.

**Comment by ilya-fedin** [comment]
Well, we need a crash report which is possible to do only using the official binary from https://desktop.telegram.org with installation of beta versions enabled in settings. But it doesn't have hardware acceleration libraries linked so I guess you won't be able to reproduce. In other words, this is blocked on the feature being not officially presented and therefore undebuggable.

**Comment by ilya-fedin** [comment]
I wonder why this report is still open. The common practice is to close bug reports unreproducible using official builds as there's nothing actionable.
